### DISCUSSION TITLE
Deployment via Intune

### DISCUSSION THREAD

**Discussion by kevbot1234** [rxhanson/Rectangle#1376] [General]
Hello i'm trying to deploy this to myself via Intune and am having trouble doing so, i'm receiving an error that the dmg does not contain the specified app, I used the id that was specified in the info in the contents of the dmg
are you familiar with this error by any chance? or could you verify whether I am using the correct one: com.knollsoft.rectangle

**Comment by rxhanson** [comment]
I'm not familiar with Intune or how it works. Not sure if it's case sensitive, so maybe try `com.knollsoft.Rectangle`?
